src/collision-validation.cc

```diff
@@ -75,19 +75,45 @@ namespace hpp {
 
       fcl::CollisionResult collisionResult;
       CollisionPairs_t::const_iterator _col;
-      if (collide (collisionPairs_, collisionRequest_, collisionResult, _col)
-          ||
-          ( checkParameterized_ &&
-            collide (parameterizedPairs_, collisionRequest_, collisionResult, _col)
-          )) {
-        CollisionValidationReportPtr_t report (new CollisionValidationReport);
-        report->object1 = _col->first;
-        report->object2 = _col->second;
-        report->result = collisionResult;
-        validationReport = report;
-        return false;
+      if(computeAllContacts_){
```

Comment:
Is it possible here to have only one loop instead of two ?
The case 'true' does the job of the case 'false' + some extraJob.
One loop would be easier to maintain but it is not a hard constraint.

Reply:
Done in 83f6672

Reply:
That's perfect.